### ISSUE TITLE
Answers to four questions not showing after clicking on "Responses" in visualise

### ISSUE THREAD

**Issue by DavidUnderdown** [nationalarchives/DiAGRAM#59]
When reviewing #58 I noticed an issue with some questions not appearing when reviewing the answers associated with a model.

The following questions with their associated answers are not appearing (text copied from the existing shinyapps prototype):

-What percentage of your material do you have at least one additional copy of?
-For those files with an additional copy, do you ensure you always have at least 2 independent copies?
-What percentage of your digital material has a copy kept offsite? 
-Based on the Government's long term flood risk assessment, how likely is it that your safest digital storage location will experience a flood? Click <a href="https://flood-warning-information.service.gov.uk/long-term-flood-risk/postcode" target="_blank">here</a> to check your flood risk.

Obviously the click here bit should be changed as it would not pass accessibility requirements. I'd suggest rewording at as "Visit [the long term flood risk tool](https://flood-warning-information.service.gov.uk/long-term-flood-risk/postcode) to check your flood risk."

**Comment by timbrock** [comment]
> Obviously the click here bit should be changed as it would not pass accessibility requirements. I'd suggest rewording at as "Visit [the long term flood risk tool](https://flood-warning-information.service.gov.uk/long-term-flood-risk/postcode) to check your flood risk."

Do we need the link at all in the table, if we're just checking what answers were given?

**Comment by DavidUnderdown** [comment]
I guess not really

**Comment by timbrock** [comment]
I've just noticed it is there (with no link at all) in the form. So I'll fix that there, as suggested above.

**Comment by DavidUnderdown** [comment]
Thanks

**Comment by timbrock** [comment]
I've just pushed a fix to this missing rows issue and added the missing link on the [dev](https://github.com/nationalarchives/DiAGRAM/tree/dev) branch. This involved tweaking the table a little - some stuff that was in the second column has been moved to the first column. Can you check the changes I've made makes sense @DavidUnderdown?
